eos_data_distribution/ndn/dbus/base.py

```diff
@@ -244,9 +244,13 @@ def _on_request_interest_complete(self, interface, res, interest):
             logger.debug('request-interest got error: %s', error)
             if str(error).find('ETRYAGAIN') != -1:
                 return self._dbus_express_interest(interest)
+            elif error.code == 24: # Timeout
```

Comment:
i'm very much not sure how to use that API...
```
In [19]: int(Gio.DBusError.SPAWN_FORK_FAILED)
Out[19]: 24
```
that said when the error gets in i get:
```
GLib.Error: g-io-error-quark: Timeout was reached (24)
```